src/main/java/io/kestra/plugin/aws/cli/AwsCLI.java

```diff
@@ -29,26 +29,20 @@
 @Getter
 @NoArgsConstructor
 @Schema(
-        title = "Execute aws commands."
+        title = "Execute AWS commands."
 )
 @Plugin(
         examples = {
                 @Example(
-                        title = "Create a simple S3 bucket",
+                        title = "Create a simple S3 bucket.",
                         code = {
-                                "accessKeyId: \"<access-key>\"",
-                                "secretKeyId: \"<secret-key>\"",
-                                "region: \"eu-central-1\"",
```

Comment:
Please keep that part to show off how credentials should be added

Reply:
This is not present in most other AWS services. 
I am planning to add an "Authentication" section on this page: https://kestra.io/plugins/plugin-aws similar to the Authentication section present for GCP here: https://kestra.io/plugins/plugin-gcp